pages/api/hostedPagesLink/[hostedPagesLinkId].ts

```diff
@@ -0,0 +1,67 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from 'next'
+import jwt from 'jsonwebtoken'
+import { type HostedPagesLinkParams, environment } from '../../../types'
+
+type Data =
+  | {
+      stakeholderId: string
+      hostedPagesLinkId: string
+      pathwayId: string
+    }
+  | {
+      error: any
+    }
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  const { hostedPagesLinkId } = req.query as HostedPagesLinkParams
+
+  const token = jwt.sign(
+    {
+      username: environment.apiGatewayConsumerName,
+      feature: 'hosted-pages-link',
+      hosted_pages_link_id: hostedPagesLinkId,
+    },
+    environment.jwtAuthSecret,
+    {
+      issuer: environment.jwtAuthKey,
+    }
+  )
+  try {
+    const hosted_pages_link = await fetch(environment.orchestrationApiUrl, {
+      method: 'POST',
+      headers: {
+        authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        query: `
+          query HostedPagesLink($id: String!) {
```

Comment:
ah, ok, now I get it why you added this query in orchestration. Not sure I like this, we have an extra network hop now go start the session, let's have a chat